devfile-support/samples/nodejs-stack.devfile.yaml

```diff
@@ -30,29 +30,29 @@ components:
       mountSources: true
 commands:
   - exec:
-      alias: download dependencies
+      id: download dependencies
       component: nodejs
       commandLine: npm install
       workdir: ${CHE_PROJECTS_ROOT}/project/app
   - exec:
-      alias: run the app
+      id: run the app
```

Comment:
"run the app" would be the `name` in this case, no? The `id` should be something like `run` or `runApp`.

Does the spec allow whitespace in command Ids?

Reply:
Afaik we didn't define a constraint that disallow spaces in ids.
And here, since I didn't add any label, I used an id that could also be used as the label.

But sure, the usual way would probably be to use a short, obvious and easy-to-remember label, and specify the user-oriented Label in the optional `label` field.